index.html

```diff
@@ -115,6 +115,31 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
         
+        <!-- ________ *ALCUNOO* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">ALCUNO</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.github.com/" target="_blank">ALCUNOO</a>
```

Comment:
Here also "href" link should be go like this `https://github.com/ALCUNOO` . 
So please replace `https://www.github.com/` with `https://github.com/ALCUNOO` and commit your changes.
We will review it and merge it